eogrow/pipelines/export_maps.py

```diff
@@ -18,7 +18,7 @@
 from eolearn.io import ExportToTiffTask
 
 from ..core.pipeline import Pipeline
-from ..utils.map import merge_maps
+from ..utils.map import cogify_inplace, merge_tiffs
```

Comment:
I guess this means the `merge_maps` can be deleted in the utils? Perhaps you forgot to push.

Reply:
I was hesitant with doing this, because 'someone might be using it somewhere' but i think you're right, it should be deleted, since it's just two lines anyway.

Reply:
Feel free to resolve and merge after update.